builder/test-integration/spec/hypriotos-image/filesystems_spec.rb

```diff
@@ -3,7 +3,7 @@
   it { should be_loaded }
 end
 
-Specinfra::Runner.run_command('modprobe overlay')
-describe kernel_module('overlay') do
+Specinfra::Runner.run_command('modprobe overlay2')
```

Comment:
@DieterReuter I've also changed this test. If Docker 1.13.0 uses overlay2 we should check for overlay2, right?